typescript/src/scss/main.scss

```diff
@@ -6,11 +12,23 @@
     margin: 20px 0;
 
     td, th {
-      text-align: right;
+      text-align: center;
       padding: 2px 5px;
-      vertical-align: top;
       border: 0;
+      justify-content: center;
     }
+    thead, .bottom{
+      background-color: rgb(136, 206, 222);
```

Comment:
Can you use SCSS variables here rather than specifying the colours directly?

Reply:
Done!